sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -203,4 +204,9 @@ private ProxyHandler getProxyHandler(AuthorizationChallengeHandler challengeHand
                     String.format(INVALID_PROXY_MESSAGE, proxyOptions.getType())));
         }
     }
+
+    public NettyAsyncHttpClientBuilder disableBufferCopy() {
```

Comment:
Should the name be inverse and take a boolean to allow for this to be flipped back?

Reply:
The name should be `disable` as, by default, it will be enabled. I can add a param to toggle.